enterprise/server/usage/usage.go

```diff
@@ -136,3 +375,40 @@ func countsToMap(tu *tables.UsageCounts) (map[string]int64, error) {
 	}
 	return counts, nil
 }
+
+// stringMapToCounts converts a Redis hashmap containing usage counts to
+// tables.UsageCounts.
+func stringMapToCounts(h map[string]string) (*tables.UsageCounts, error) {
+	hInt64 := map[string]int64{}
```

Comment:
it is common in go to use make() to initialize maps -- especially in cases where you know the number of entries you plan to insert. This allows you program to make one allocation of the right size, rather than having to dynamically resize the map as you insert more and more items.

hInt64  := make(map[string]int64{}, len(h))

Reply:
Knew about `make`, just forgot to use it here. Done